src/components/LeafletMap/style.scss

```diff
@@ -0,0 +1,181 @@
+.map-component {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 100;
+
+  .leaflet-container {
+    min-height: 100vh;
+  }
+
+  #mapdiv {
+    position: fixed;
+    width: 100%;
+    height: 100vh;
+    top: 0;
+    left: 0;
+  }
+
+  .embed-search {
+    max-width: 328px;
+
+    @media screen and (max-width: 576px) {
+      max-width: 100%;
+    }
+
+    .invoer {
+      display: flex;
+      box-shadow: 2px 2px 0 0 #999;
+    }
+
+    input[type='text'] {
+      height: 32px;
+      border: solid 1px #ccc;
+      box-shadow: none;
+      padding-top: 0;
+      padding-bottom: 0;
+      line-height: 32px;
+    }
+
+    &__button {
+      background-color: #f6f6f6 !important;
+      border: solid 1px #ccc !important;
+      border-left: none;
+
+      &:hover,
+      &:focus {
+        background-color: #ccc !important;
+        border: solid 1px #000 !important;
+        color: #000 !important;
+        fill: #000;
+      }
+
+      &.primary.action:before {
+        content: url(../../images/icon-search-black.svg);
+        display: block;
+        height: 24px;
+        left: 50%;
+        position: absolute;
+        top: 55%;
+        -webkit-transform: translate(-50%, -50%);
+        transform: translate(-50%, -50%);
+        width: 24px;
+      }
+    }
+
+    &__auto-suggest {
+      width: 100%;
+    }
+  }
+
+  &__loading {
+    background-color: rgba(0, 0, 0, 0.2);
+    display: block;
+    width: calc(100% - 30px); // padding left and right of grid
+    height: 100%;
+    position: absolute;
+    z-index: 1001;
+  }
+}
+
+.about-button {
+  padding: 8px;
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  box-shadow: 2px 2px 0 0 #999999;
+  background-color: #f6f6f6;
+  border: solid 1px #cccccc;
+  cursor: pointer;
+  z-index: 1010;
+
+  font-family: 'AvenirNextLTW01-Regular';
+  font-size: 16px;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1;
+  letter-spacing: normal;
+  color: #000000;
+
+  &:hover,
+  &:focus {
+    background-color: #ccc;
+    border-color: #000 !important;
+    color: #000 !important;
+    fill: #000;
+  }
+
+  @media screen and (max-width: 540px) {
+    left: 10px;
+    top: auto;
+    right: auto;
+    bottom: 20px;
+  }
+}
+
+// Override cluster design from leaflet
+.marker-cluster {
+  color: white;
+  background-color: white !important;
+  box-shadow: 1px 1px 1px #666666;
+
+  div {
```

Comment:
Try to prevent using element selectors; those are way too generic. Can you use class names instead?

Reply:
Apparently these css rules aren't even used right now. removing them